google/cloud/storagecontrol/v2/internal/storage_control_tracing_stub.cc

```diff
@@ -37,6 +37,7 @@ StorageControlTracingStub::CreateFolder(
     google::storage::control::v2::CreateFolderRequest const& request) {
   auto span = internal::MakeSpanGrpc("google.storage.control.v2.StorageControl",
                                      "CreateFolder");
+  span->SetAttribute("gl-cpp.request_id", request.request_id());
```

Comment:
ideally this attribute is set when the span is created (so it can be used for sampling) but no big deal.